airflow/models/variable.py

```diff
@@ -47,7 +48,7 @@ class Variable(Base, LoggingMixin):
 
     id = Column(Integer, primary_key=True)
     key = Column(String(ID_LEN), unique=True)
-    _val = Column('val', Text)
+    _val = Column('val', Text().with_variant(MEDIUMTEXT, 'mysql'))
```

Comment:
Why do some of the Text fields need a MySQL variant, but some don’t? Should we unify them?

Reply:
Same here. MySQL was changed to MEDIUMTEXT in the DB: https://github.com/apache/airflow/blob/main/airflow/migrations/versions/0026_1_8_2_increase_text_size_for_mysql.py

Reply:
Looks like this is done by a case-by-case basis; use `Text` by default, and change it if that’s not big enough. Let’s keep this as-is then. A later PR to unify these types (like how we do `StringID` would be a good idea.